packages/asset-swapper/src/utils/market_operation_utils/sampler_operations.ts

```diff
@@ -1733,6 +1765,14 @@ export class SamplerOperations {
                             takerFillAmounts,
                         );
                     }
+                    case ERC20BridgeSource.WOOFi: {
+                        return this.getWOOFiSellQuotes(
```

Comment:
If i understand correctly, woofi only works with a set of 5-6 tokens. If `takerToken` or `makerToken` isn't in that set, it may make sense to return an empty array immediately.

The same applies to `getWOOFiBuyQuotes` below.